### ISSUE TITLE
Special characters in character names lead to broken character sheets.

### ISSUE THREAD

**Issue by LawOfTheLand** [RolePlayGateway/roleplaygateway#62]
https://www.roleplaygateway.com/roleplay/the-multiverse/characters/ulfhe%C3%B0nar

The Unicode is parsed correctly in the URL, but the rest of the page breaks.

**Comment by colearkenach** [comment]
Fiddled around with [character:{numericID}] to figure out this specific character's ID was "255873". I wonder if there's a way to redirect to a url using the character's ID if their username url doesn't work?

Figured that might be relevant, since I've seen some other urls that used special characters break.

**Comment by lostamongtrees** [comment]
Can confirm

![image](https://user-images.githubusercontent.com/52435234/60764734-018e0c80-a05d-11e9-9d0e-f833724e4b89.png)

![image](https://user-images.githubusercontent.com/52435234/60764740-1c608100-a05d-11e9-98ca-6d949b6099de.png)

**Comment by lostamongtrees** [comment]
Thanks @LawOfTheLand - closing this issue as I compiled it into #124 as its part of a larger issue regarding special characters and *things*
